packages/patternfly-4/react-core/src/components/EmptyState/EmptyStateIcon.tsx

```diff
@@ -21,12 +21,21 @@ export interface EmptyStateIconProps extends IconProps {
   className?: string;
   /** Icon component to be rendered inside the EmptyState */
   icon: string | React.FunctionComponent<IconProps>;
+  isWrapped?: boolean;
 }
 
 export const EmptyStateIcon: React.FunctionComponent<EmptyStateIconProps> = ({
   className = '',
   icon: IconComponent,
+  isWrapped = false,
   ...props
-}: EmptyStateIconProps) => (
-  <IconComponent className={css(styles.emptyStateIcon, className)} {...props} aria-hidden="true" />
-);
+}: EmptyStateIconProps) => {
+  const classNames = css(styles.emptyStateIcon, className);
+  return isWrapped ? (
+    <div className={classNames}>
+      <IconComponent {...props} aria-hidden="true" />
+    </div>
```

Comment:
Will this allow you to put a the `<Spinner>` component in the empty state as the "icon"?

Reply:
I just realized that it's a little complicated. Is `<Spinner>` imported from `react-icons` or is a separate component?

Reply:
It’s a separate component.

Reply:
It’s a separate component.